lib/plugins/Plugins.json

```diff
@@ -46,6 +46,7 @@
     "./aws/package/compile/events/cloudWatchEvent/index.js",
     "./aws/package/compile/events/cloudWatchLog/index.js",
     "./aws/package/compile/events/cognitoUserPool/index.js",
+    "./aws/package/compile/events/sqs/index.js",
```

Comment:
I was wondering why this plugin wasn't running!

Reply:
🙈 Yea, took me a while to find this one 😁